go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/elazarl/goproxy/ext v0.0.0-20190421051319-9d40249d3c2f // indirect
 	github.com/fatih/camelcase v1.0.0 // indirect
 	github.com/ghodss/yaml v1.0.1-0.20180820084758-c7ce16629ff4
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/go-logr/logr v0.1.0
```

Comment:
did you run `go mod tidy`? Are you sure that add this libs are required?

Reply:
The changes to go.mod and the Makefile were unintentional. Ill remove them in the next commit.